### PR TITLE
Add subdomain as an env variable

### PR DESCRIPTION
**Pull request by Sivli-Embir**
This adds Account Subdomain to the CY job env

We now require subdomains to be set on all of the apps domains. The old apps general subdomain is now no longer usable and web-root will auto-redirect users to subdomains. E2E before this did not have that data and was redirecting to undefined. This redirect was causing issues, including import map override.

### CODE REVIEW

File: .github/workflows/standard_spa_pr_NPM.yml [hunk 1]
```diff
@@ -4,11 +4,10 @@ on:
   workflow_call:
     inputs:
       working_directory:
-        description:
-          'The working directory from which to run the workflow commands'
+        description: "The working directory from which to run the workflow commands"
```
Comment (Sivli-Embir) [inline .github/workflows/standard_spa_pr_NPM.yml:7]: I installed the official GH VS CODE actions plugin and it reformated the full page

File: .github/workflows/standard_spa_pr_NPM.yml [hunk 2]
```diff
@@ -300,24 +298,24 @@ jobs:
     outputs:
       artemisAccountName: ${{ steps.accountInfo.outputs.artemisAccountName }}
       artemisAccountId: ${{ steps.accountInfo.outputs.artemisAccountId }}
+      artemisAccountSubdomain: ${{ steps.accountInfo.outputs.artemisAccountSubdomain }}
```
Comment (Sivli-Embir) [inline .github/workflows/standard_spa_pr_NPM.yml:301]: Added this line

File: .github/workflows/standard_spa_pr_NPM.yml [hunk 3]
```diff
@@ -327,6 +325,7 @@ jobs:
         # Use jq to extract the accountId and user sessions from the json file in a raw form
         run: |
           echo "artemisAccountName=$(jq -r .[0].metadata.accountName < artemis-run.json)" >> $GITHUB_OUTPUT
+          echo "artemisAccountSubdomain=$(jq -r .[0].metadata.accountSubdomain < artemis-run.json)" >> $GITHUB_OUTPUT
```
Comment (Sivli-Embir) [inline .github/workflows/standard_spa_pr_NPM.yml:328]: Added this line

File: .github/workflows/standard_spa_pr_NPM.yml [hunk 4]
```diff
@@ -415,6 +414,7 @@ jobs:
           REPO_NAME: ${{ inputs.external_pr_repo_name || github.event.repository.name }}
           SPEC: ${{ inputs.spec_to_run }}
           ACCOUNT: ${{ needs.prepare_for_e2e_test.outputs.artemisAccountName }}
+          ACCOUNT_SUBDOMAIN: ${{ needs.prepare_for_e2e_test.outputs.artemisAccountSubdomain }}
```
Comment (Sivli-Embir) [inline .github/workflows/standard_spa_pr_NPM.yml:417]: Added this line

**Review by jablonnc — APPROVED** [review]

**Review by ethan-steinman-j1 — APPROVED** [review]

**Review by behobu — APPROVED** [review]

**Pull request merged**